src/main/java/seedu/address/logic/commands/PaidPersonWhoOwedCommand.java

```diff
@@ -54,10 +54,10 @@ public CommandResult execute(Model model) throws CommandException {
         String string = personToRemove.toString();
         String name = string.split(";")[0];
         String amount = string.split(":")[1].split(" ")[1];
-        String personRemoved = sb.append(name).append(" Amount: $").append(amount).toString();
+        String personRemoved = sb.append(name).append(amount).toString();
```

Comment:
What is the point of this string as it is not used.

Reply:
i was using it before, not anymore

Reply:
Think if you are not using the code, please remove the line.

Reply:
done